sdk/attestation/attestation/test/utils/helpers.ts

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { assert } from "chai";
+
+import { AttestationClient } from "../../src/";
+
+// import { decode } from "jsonwebtoken";
+import * as jsrsasign from "jsrsasign"; // works in the browser
+
+import { decode } from "./decodeJWT";
+
+export function decodeJWT(
+  attestationToken: string,
+  client: AttestationClient
+): {
+  [key: string]: any;
+} {
+  const decoded = decode(attestationToken, { complete: true, json: true });
+  if (decoded?.header) {
+    assert.notEqual(decoded.header.alg, "none");
+    assert.equal(decoded.header.typ, "JWT");
+    assert.equal(decoded.header.jku, client.instanceUrl + "/certs");
+    return decoded;
+  }
+  throw new Error(`decoded token did not have header: ${decoded}`);
+}
+
+export async function verifyAttestationToken(
+  attestationToken: string,
+  client: AttestationClient
+): Promise<{
+  [key: string]: any;
+}> {
+  const decoded = decodeJWT(attestationToken, client);
+  const keyId = decoded?.header.kid;
+
+  const signingCerts = await client.signingCertificates.get();
+  let signingCertx5C;
+  if (signingCerts?.keys) {
+    assert(signingCerts.keys?.length > 0);
+    for (const key of signingCerts.keys) {
+      if (key.kid === keyId) {
+        signingCertx5C = key.x5C;
+      }
+    }
+    if (signingCertx5C !== null && signingCertx5C !== undefined) {
+      // Convert the inbound certificate to PEM format so the verify function is happy.dir dist
```

Comment:
```suggestion
      // Convert the inbound certificate to PEM format so the verify function is happy
```

Reply:
I will leave this module as is because it was copied from somewhere else.

Reply:
Actually this is code I wrote :). THe dir dist was probably a typo I left in. This is the code I wrote to turn the attestation validation certificate (signingCertx5C) into a PEM encoded string (it's not correctly formatted, but apparently good enough) to be passed in to create an RSA key.